plugins/procfs/README.md

```diff
@@ -1,3 +1,153 @@
 # Procfs plugin
 
-Collects processes- and system-related metrics by reading the `/proc` virtual filesystem.
+Collects processes and system-related metrics by reading the [proc](<https://www.kernel.org/doc/html/latest/filesystems/proc.html>) virtual filesystem on Linux based operating systems.
+
+## Requirements
+
+- Linux operating system.
+- Read access to the `/proc` virtual file system. Depending on the mount options, [some privileges might be needed](#procfs-access).
+
+## Metrics
+
+There are various information collected by this plugin relative to Kernel, CPU, memory and processes :
+
+|Name|Type|Unit|Description|Resource|ResourceConsumer|Attributes|
+|----|----|----|-----------|--------|----------------|----------|
+|`kernel_cpu_time`|CounterDiff|millisecond|Time during the CPU is busy|LocalMachine|LocalMachine|[cpu_state](#cpu_state)|
+|`kernel_context_switches`|CounterDiff|none|Number of context switches : Operation allowing a single CPU to manage multiple processes efficiently, involves saving the state of a currently running process and loading the state of another process, enabling multitasking and optimal CPU utilization|LocalMachine|LocalMachine||
+|`kernel_new_forks`|CounterDiff|none|Number of forked operations : When a process creates a copy of itself|LocalMachine|LocalMachine||
+|`kernel_n_procs_running`|Gauge|none|Number of processes in a runnable state|LocalMachine|LocalMachine||
+|`kernel_n_procs_blocked`|Gauge|none|Numbers of processes that are blocked on input/output operations|LocalMachine|LocalMachine||
+|`cpu_time_delta`|CounterDiff|nanosecond|CPU usage|LocalMachine|Process|[kind](#kind)|
+|`memory_usage`|Gauge|bytes|Memory usage|LocalMachine|Process|[kind](#kind)|
+
+And in case we configure the plugin with the collect of detailed memory metrics :
+
+|Name|Type|Unit|Description|Resource|ResourceConsumer|Attributes|
+|----|----|----|-----------|--------|----------------|----------|
+|`mem_total`|Gauge|bytes|Total amount of physical memory available in the system|LocalMachine|LocalMachine||
+|`mem_free`|Gauge|bytes|Refers to truly free memory, which is not used by processes, caches or buffers|LocalMachine|LocalMachine||
+|`mem_available`|Gauge|bytes|More accurate estimate of the actual available memory, taking into account free memory and also caches and buffers that can be freed|LocalMachine|LocalMachine||
+|`cached`|Gauge|bytes|Cached is the memory used for ROM cache (page cache) in memory|LocalMachine|LocalMachine||
+|`swap_cached`|Gauge|bytes|Memory that was placed in swap and brought back to RAM, but remains in swap file|LocalMachine|LocalMachine||
+|`active`|Gauge|bytes|Memory that has been used recently|LocalMachine|LocalMachine||
+|`inactive`|Gauge|bytes|Used memory that can be reclaimed, freed, or moved to swap.|LocalMachine|LocalMachine||
+|`mapped`|Gauge|bytes|Memory used for files associated with processes in memory, file and memory mapping.|LocalMachine|LocalMachine||
```

Comment:
If this is about `plugins.procfs.memory.metrics`, this table lacks some values. But don't copy the whole doc, it's better to remove the table and explain that the metrics come from `/proc/meminfo`, with their names turned into snake_case (e.g. `MemTotal` becomes `mem_total`). Add a link to https://man7.org/linux/man-pages/man5/proc_meminfo.5.html.